hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,64 @@
+RELEASE_TYPE: major
+
+Welcome to the next major version of Hypothesis!
+
+There are no new features here, as we release those in minor versions.
+Instead, 5.0 is a chance for us to remove deprecated features (many already
+converted into no-ops), and turn a variety of warnings into errors.
+
+If you were running on the last version of Hypothesis 4.x *without any
+Hypothesis deprecation warnings*, this will be a very boring upgrade.
+**In fact, nothing will change for you at all.**
+
+.. note::
+    This release drops support for Python 2, which has passed
+    `its end of life date <https://devguide.python.org/#status-of-python-branches>`__.
+    The `Python 3 Statement <https://python3statement.org/>`__ outlines our
+    reasons, and lists many other packages that have made the same decision.
+
+    ``pip install hypothesis`` should continue to give you the latest compatible version.
+    If you have somehow ended up with Hypothesis 5.0 on Python 2, you need to update your
+    packaging stack to ``pip >= 9.0`` and ``setuptools >= 24.2`` - see `here for details
+    <https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires>`__.
```

Comment:
Is there a missing step here? `Then run "pip install hypothesis<5"`? Just updating pip and setuptools won't magically fix your broken Hypothesis install.